litert/vendors/intel_openvino/dispatch/BUILD

```diff
@@ -34,7 +34,6 @@ litert_dynamic_lib(
     ],
     copts = [
         "-Os",
-        "-fno-exceptions",
```

Comment:
Thanks, please add `"-fexceptions",` to the `copts` of the targets under `litert/vendors/intel_openvino/...` so that for the google-internal build exceptions are enabled.

Reply:
Done